src/components/AirdropModal/index.tsx

```diff
@@ -0,0 +1,308 @@
+            setIsClaimAvailable(true)
+          }
+        } catch (err) {
+          displayError()
+        }
+      })()
+    }
+  }, [account, contract, provider, setIsClaimAvailable])
+
+  const makeClaim = async () => {
+    try {
+      if (contract && claim && claim.amount && claim.merkleProof && provider) {
+        setIsSubmitting(true)
+
+        await contract
+          .connect(provider?.getSigner())
+          .functions.claim(claim.index, account, claim?.amount, claim?.merkleProof)
+        setIsSubmitting(false)
+        setIsClaimed(true)
+        setIsClaimAvailable(false)
+      }
+    } catch (err) {
+      setIsSubmitting(false)
+      displayError()
+    }
+  }
+
+  return (
+    <>
+      <Modal hideBorder isOpen={isOpen} onDismiss={usdcAirdropToggle} maxHeight={90} maxWidth={400}>
+        <ModalWrap>
+          {isClaimed ? (
+            <ClaimContainer>
+              <ThemedText.HeadlineSmall>Congratulations!</ThemedText.HeadlineSmall>
+              <SuccessText>
+                You have successfully claimed {totalAmount} USDC. Thank you for supporting Genie.xyz.
+              </SuccessText>
+              <EtherscanLink href="https://etherscan.io/" target="_blank">
+                <ThemedText.Link>
+                  <div style={{ display: 'flex', alignItems: 'center', textAlign: 'center', justifyContent: 'center' }}>
+                    <span style={{ marginRight: 8 }}>Etherscan</span>
+                    <ChevronRightIcon />
+                  </div>
+                </ThemedText.Link>
+              </EtherscanLink>
+
+              <CloseButton size={ButtonSize.medium} emphasis={ButtonEmphasis.medium} onClick={usdcAirdropToggle}>
+                Close
+              </CloseButton>
+            </ClaimContainer>
+          ) : (
+            <>
+              <ImageContainer>
+                <TextContainer>
+                  <SyledCloseIcon onClick={usdcAirdropToggle} stroke="white" />
+                  <MainHeader>Uniswap NFT Airdrop</MainHeader>
+                  <USDCLabel>{totalAmount} USDC</USDCLabel>
+                  <Line />
+                  <RewardsDetailsContainer>
+                    <RewardsText>Trading rewards</RewardsText>{' '}
+                    <CurrencyText>{totalAmount === 300 || totalAmount === 1300 ? '300 USDC' : '0'}</CurrencyText>
+                  </RewardsDetailsContainer>
+                  <RewardsDetailsContainer>
+                    <RewardsText>Genie NFT holder rewards</RewardsText>{' '}
+                    <CurrencyText>{totalAmount > 300 ? `1000 USDC` : '0'}</CurrencyText>
+                  </RewardsDetailsContainer>
+                </TextContainer>
+                <StyledImage src={airdropBackgroundv2} />
+              </ImageContainer>
+              <Body>
+                <RewardsInformationText>
+                  As a long time supporter of Genie, you’ve been awarded {totalAmount} USDC tokens. Read more about
```

Comment:
I believe there was a ticket to remove the text `Read more about Uniswap NFT` since that's repeated in the link below